### PR TITLE
Pep8ize ii 4

### PR DESCRIPTION
**Pull request by rob-baron**
These are odds and ends that didn't fit in well

### CODE REVIEW

**Review by kamfonik — APPROVED** [review]
Looks good.

File: atmosphere/settings/__init__.py [hunk 1]
```diff
@@ -553,7 +553,7 @@ def generate_secret_key(secret_key_path):
 """
 Import local settings specific to the server, and secrets not checked into Git.
 """
-from atmosphere.settings.local import *
```
Comment (knikolla) [inline atmosphere/settings/__init__.py]: These are not equivalent.

File: core/models/application_version.py [hunk 2]
```diff
@@ -360,21 +360,21 @@ def create_app_version(
     last_version = app.latest_version
     if last_version:
         # DEFAULT: Use kwargs.. Otherwise: Inherit information from last
-        if change_log != None:
+        if change_log is not None:
```
Comment (knikolla) [inline core/models/application_version.py:363]: This can be rewritten as ``if change_log``. (optional)

File: core/models/application_version.py [hunk 3]
```diff
@@ -360,21 +360,21 @@ def create_app_version(
...
+        if allow_imaging is not None:
             app_version.allow_imaging = allow_imaging
         else:
             app_version.allow_imaging = last_version.allow_imaging
         app_version.save()
         transfer_licenses(last_version, app_version)
         transfer_membership(last_version, app_version)
     else:
-        if change_log == None:
+        if change_log is None:
```
Comment (knikolla) [inline core/models/application_version.py:375]: ``if not change_log``. (optional)

**Review by knikolla — CHANGES_REQUESTED** [review]

File: atmosphere/settings/__init__.py (hunk 1 above)
Comment (kamfonik) [reply atmosphere/settings/__init__.py]: Oh, good catch.  I think the same would apply to `api/__init__.py` (file 2 above)?

File: api/v1/views/identity_membership.py [hunk 4]
```diff
@@ -89,16 +90,16 @@ def delete(self, request, provider_uuid,
             return failure_response(
                 status.HTTP_404_NOT_FOUND,
                 "Identity does not exist.")
-        if not identity.can_share(user):
+        if not identity.can_share(request.user):
```
Comment (lucashxu) [inline api/v1/views/identity_membership.py:93]: In L28, " user = request.user" So you don't need a `request` here

File: api/v1/views/identity_membership.py [hunk 5]
```diff
@@ -89,16 +90,16 @@ def delete(self, request, provider_uuid,
...
             logger.error(
                 "User %s cannot remove sharing from identity %s. "
                 "This incident will be reported"
-                % (user, identity_uuid))
+                % (request.user, identity_uuid))
```
Comment (lucashxu) [inline api/v1/views/identity_membership.py:97]: ditto

File: api/v1/views/identity_membership.py [hunk 6]
```diff
@@ -89,16 +90,16 @@ def delete(self, request, provider_uuid,
...
             return failure_response(
                 status.HTTP_401_UNAUTHORIZED,
                 "User %s cannot remove sharing from identity %s. "
                 "This incident will be reported"
-                % (user, identity_uuid))
+                % (request.user, identity_uuid))
```
Comment (lucashxu) [inline api/v1/views/identity_membership.py:102]: ditto

File: api/v2/serializers/post/instance.py [hunk 7]
```diff
@@ -5,6 +5,7 @@
 from rest_framework import serializers
 from rest_framework.exceptions import ValidationError
 from api.v2.serializers.fields.base import ReprSlugRelatedField
+from Atmosphere import settings
```
Comment (lucashxu) [inline api/v2/serializers/post/instance.py:8]: L63 is using `settings` currently.

File: service/tasks/monitoring.py [hunk 8]
```diff
@@ -493,25 +493,6 @@ def get_shared_identities(account_driver, cloud_machine, tenant_id_name_map):
     return identity_list
 
 
-def update_membership(application, shared_identities):
```
Comment (lucashxu) [inline service/tasks/monitoring.py:496]: why we take the whole function out?

**Comment by xuhang57** [comment]
Why you merge this?

File: api/v1/views/identity_membership.py (hunk 4 above)
Comment (jeremyfreudberg) [reply api/v1/views/identity_membership.py:93]: L28 is in a different class

Comment (rob-baron) [reply api/v1/views/identity_membership.py:93]: Yes, I had the choice between defining an alias to request.user (as is done in L28) or just using request.  I choose the latter as for 2 function calls.  As jeremy points out L28 is in a different scope that is not accessible from here, so yes you do need something here.

Comment (lucashxu) [reply api/v1/views/identity_membership.py:93]: You guys are right. But if you see L28 and L55. so I believe we should do it consistently.

**Review by xuhang57 — COMMENTED** [review]
Ok. so what is the difference between this PR and the `5`?

**Pull request merged**